front/tests/operational-studies.spec.ts

```diff
@@ -0,0 +1,65 @@
+import { test, expect } from '@playwright/test';
+import { PlaywrightHomePage } from './home-page-model';
+
+// Describe the test suite for the home page of OSDR
```

Comment:
We don’t need to have this comment here, the `test.describe` gives enough context as to what it’s doing